doc/classes/SpringBoneSimulator3D.xml

```diff
@@ -0,0 +1,608 @@
+			<description>
+				Returns [code]true[/code] if the config can be edited individually for each joint.
+			</description>
+		</method>
+		<method name="is_end_bone_extended" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="index" type="int" />
+			<description>
+				Returns [code]true[/code] if the end bone is extended to have the tail.
+			</description>
+		</method>
+		<method name="set_center_bone">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="bone" type="int" />
+			<description>
+				Sets the center bone index of the bone chain.
+			</description>
+		</method>
+		<method name="set_center_bone_name">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="bone_name" type="String" />
+			<description>
+				Sets the center bone name of the bone chain.
+				The specified center bone origin is treated as the origin for calculating the wiggle.
+				The farther the joint is from the center, the stronger the wiggle will be when moving.
+			</description>
+		</method>
+		<method name="set_collision_count">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="count" type="int" />
+			<description>
+				Sets the number of collisions in the collision list at [param index] in the settings when [method are_all_child_collisions_enabled] is [code]false[/code].
+			</description>
+		</method>
+		<method name="set_collision_path">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="collision" type="int" />
+			<param index="2" name="node_path" type="NodePath" />
+			<description>
+				Sets the node path of the [SpringBoneCollision3D] at [param collision] in the bone chain's collision list when [method are_all_child_collisions_enabled] is [code]false[/code].
+			</description>
+		</method>
+		<method name="set_drag">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="drag" type="float" />
+			<description>
+				Sets the drag force of the bone chain. The greater the value, the more suppressed the wiggling.
+				The value is scaled by [method set_drag_damping_curve] and cached in each joint setting in the joint list.
+			</description>
+		</method>
+		<method name="set_drag_damping_curve">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="curve" type="Curve" />
+			<description>
+				Sets the drag force damping curve of the bone chain.
+			</description>
+		</method>
+		<method name="set_enable_all_child_collisions">
+			<return type="void" />
+			<param index="0" name="index" type="int" />
+			<param index="1" name="enabled" type="bool" />
+			<description>
+				If sets [param enabled] to [code]true[/code], the all child [SpringBoneCollision3D]s are contained in the collision list at [param index] in the settings.
```

Comment:
This sentence doesn't make sense, it looks like a copy-paste of the `get` version but doesn't work